.cirrus.yml

```diff
@@ -17,7 +17,7 @@ env:
     # Save a little typing (path relative to $CIRRUS_WORKING_DIR)
     SCRIPT_BASE: "./contrib/cirrus"
     FEDORA_NAME: "fedora-38"
```

Comment:
```suggestion
    FEDORA_NAME: "fedora-39ß"
```

Reply:
I think this var isn't used anywhere should we just remove it from the yaml?

Reply:
I mean the image suffix already contains the distro matrix anyway.

Reply:
Yeah, it's entirely cosmetic in the task names, I'm cool to just remove it.  Wasn't sure if that was okay or not.  I'll remove it.